psydac/linalg/stencil.py

```diff
@@ -402,6 +407,23 @@ def _toarray_parallel_with_pads( self ):
 #        return out.flatten()
         return out.reshape(-1)
 
+    def topetsc( self ):
+        space = self.space
+        assert space.parallel
```

Comment:
I think it would be very useful if the user could use the PETSc data structure and solvers also in the serial case. Can this be done?

Reply:
I assumed that everything exist in scipy in the serial case, and the user can always call it with one process in the parallel case , but probably yes it can be done,  it needs a little bit of work, because I use the information provided by the cart in ordrer for me to create the petsc cart

Reply:
Please create a separate issue: "make petsc backend work in serial".

Reply:
Thanks for creating #79